geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/Dockerfile_tmpl

```diff
@@ -1,4 +1,7 @@
-FROM camptocamp/geomapfish-tools:{{geomapfish_version}} as builder
+ARG {{geomapfish_version_tag}}
```

Comment:
Not sure to understand why we need a variable here, it seems it's always "GEOMAPFISH_VERSION"

Reply:
:- ) Just for coherence between the tow images :-)

Reply:
Sorry but I do not see where they are used, here or in other dockerfile.